x-pack/plugins/observability/public/components/slo/burn_rate/alert_details/components/alerts_history/alerts_history_panel.tsx

```diff
@@ -0,0 +1,206 @@
+import { useKibana } from '../../../../../../utils/kibana_react';
+import { ErrorRateChart } from '../../../../error_rate_chart';
+import { BurnRateAlert, BurnRateRule } from '../../alert_details_app_section';
+import { getActionGroupFromReason } from '../../utils/alert';
+
+interface Props {
+  slo?: GetSLOResponse;
+  alert: BurnRateAlert;
+  rule: BurnRateRule;
+  isLoading: boolean;
+}
+
+export function AlertsHistoryPanel({ rule, slo, alert, isLoading }: Props) {
+  const {
+    services: { http },
+  } = useKibana();
+  const { isLoading: isAlertsHistoryLoading, data } = useAlertsHistory({
+    featureIds: ['slo'],
+    ruleId: rule.id,
+    dateRange: {
+      from: 'now-30d',
+      to: 'now',
+    },
+    http,
+  });
+
+  const actionGroup = getActionGroupFromReason(alert.reason);
+  const actionGroupWindow = (
+    (alert.fields[ALERT_RULE_PARAMETERS]?.windows ?? []) as WindowSchema[]
+  ).find((window: WindowSchema) => window.actionGroup === actionGroup);
+  const dataTimeRange = {
+    from: moment().subtract(30, 'day').toDate(),
+    to: new Date(),
+  };
+
+  function getAlertsLink() {
+    const kuery = `kibana.alert.rule.uuid:"${rule.id}"`;
+    return http.basePath.prepend(`/app/observability/alerts?_a=${rison.encode({ kuery })}`);
+  }
+
+  if (isLoading) {
+    return <EuiLoadingChart size="m" mono data-test-subj="loading" />;
+  }
+
+  if (!slo) {
+    return null;
+  }
+
+  return (
+    <EuiPanel paddingSize="m" color="transparent" hasBorder data-test-subj="alertsHistoryPanel">
+      <EuiFlexGroup direction="column" gutterSize="m">
+        <EuiFlexGroup direction="column" gutterSize="none">
+          <EuiFlexGroup direction="row" justifyContent="spaceBetween">
+            <EuiFlexItem grow={false}>
+              <EuiTitle size="xs">
+                <h2>
+                  {i18n.translate(
+                    'xpack.observability.slo.burnRateRule.alertDetailsAppSection.alertsHistory.title',
+                    { defaultMessage: '{sloName} alerts history', values: { sloName: slo.name } }
+                  )}
+                </h2>
+              </EuiTitle>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiLink color="text" href={getAlertsLink()} data-test-subj="alertsLink">
+                <EuiIcon type="sortRight" style={{ marginRight: '4px' }} />
+                <FormattedMessage
+                  id="xpack.observability.slo.burnRateRule.alertDetailsAppSection.alertsHistory.alertsLink"
+                  defaultMessage="View alerts"
+                />
+              </EuiLink>
+            </EuiFlexItem>
+          </EuiFlexGroup>
+
+          <EuiFlexItem grow={false}>
+            <EuiText size="s" color="subdued">
+              <span>
+                {i18n.translate(
+                  'xpack.observability.slo.burnRateRule.alertDetailsAppSection.alertsHistory.subtitle',
+                  {
+                    defaultMessage: 'Last 30 days',
```

Comment:
Similar to my question above. Is it going to always be last 30 days or is it dynamic and depends on the SLO time window definition?